.github/workflows/ci.yml

```diff
@@ -206,7 +206,7 @@ jobs:
         uses: ./.github/actions/run-script
         if: github.event_name != 'pull_request' && github.ref == 'refs/heads/main'
         with:
-          script-name: "test:visual:ci --filter='./packages/components/*'"
+          script-name: "test:visual:ci --filter='{./packages/components/*}[HEAD^1]'"
```

Comment:
Using a variation of the following syntax - https://turbo.build/repo/docs/core-concepts/monorepos/filtering

```
# Test each workspace in the '@scope' scope that
# is in the 'packages' directory, if it has
# changed in the last commit
turbo run test --filter=@scope/*{./packages/*}[HEAD^1]